test/MultiAssembly.ByAssembly.docsproj

```diff
@@ -1,11 +1,15 @@
-<Project Sdk="CloudNimble.DotNetDocs.Sdk/1.0.0">
+<Project>
 
+  <Import Project="..\src\CloudNimble.DotNetDocs.Sdk\Sdk\Sdk.props" />
```

Comment:
Here's an example of how to test local SDKs. An SDK is just a props import at the logical top of the file, and a targets import at the logical bottom of the file. So you can do that yourself and get pretty much the same experience.